Azure-Samples/Cognitive-Services-Voice-Assistant
# updated tv image

## Purpose
This PR updates the TV image to a more generic one.

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x ] Other... Please describe:
content change to update TV image

## How to Test
1. Grab the code, deploy per instructions and check.
2. Alternatively, you can take this new image and replace your current image in your pre-deployed blob storage for this demo

## What to Check
Verify that the new image is performing similar to the previous one, there is no code change here at all, just swap of the png file.
